portindex2json/portindex2json.tcl

```diff
@@ -91,4 +91,13 @@ while {[gets $fd line] >= 0} {
 }
 
 chan configure stdout -encoding utf-8
-puts [::json::write array {*}$objects]
+if {$argc == 2} {
+    # If a commit hash is provided then the output is a JSON object with keys "commit" and "ports".
+    # Value of "ports" is a list of JSON objects, each object containing info for one port.
+    set json_output(commit) [::json::write string [lindex $argv 1]]
```

Comment:
I would make this one level deeper just in case. Maybe something like
```json
{
    "info": {
        "commit": "<shasum>",
    },
    "ports": ["..."],
}
```

Reply:
Thank you, I have used this format now.